### PR TITLE
enhancement #573 - Start using profile v2 APIs in the FTU flow

### LINKED ISSUE #573: Start using profile v2 APIs in the FTU flow
Update Timezone dropdown to accept options to be populated with

### PR DESCRIPTION
**Pull request by jasper-dassana**
Closes #573

### CODE REVIEW

File: src/components/Timezone/types.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { SelectOption } from '../Select/SingleSelect'
+
+export interface TimezoneSelectOption extends SelectOption {
```
Comment (nancy-dassana) [inline src/components/Timezone/types.ts:3]: What is this for?

Comment (jasper-dassana) [reply src/components/Timezone/types.ts]: It's unnecessary - I'll remove it

**Review by nancy-dassana — APPROVED** [review]
LGTM

**Review by nancy-dassana — APPROVED** [review]
LGTM

**Review by nancy-dassana — APPROVED** [review]
LGTM

**Pull request merged**